applicationinsights.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This change was needed as part of pulling in the latest dps gradle spring boot plugin.